contracts/Committee.sol

```diff
@@ -175,44 +175,68 @@ contract Committee is ICommittee, ContractRegistryAccessor {
 
 	/// @dev Called by: Elections contract
 	/// Returns the committee members and their weights
-	function _getCommittee() public view returns (address[] memory addrs, uint256[] memory weights) {
-		(Participant[] memory members,) = loadParticipants(participants, NullMember());
-		Participant[] memory committee = computeCommittee(members, settings).committee;
-		return toAddrsWeights(committee);
+	function _getCommittee() public view returns (address[] memory addrs, uint256[] memory weights, bool[] memory compliance) {
+		// todo in case of committeeInfo.pendingChanges, this is stale
+		CommitteeInfo memory _committeeInfo = committeeInfo;
+		uint bitmap = uint(_committeeInfo.committeeBitmap);
+		uint committeeSize = _committeeInfo.committeeSize;
+
+		addrs = new address[](committeeSize);
+		weights = new uint[](committeeSize);
+		compliance = new bool[](committeeSize);
+		bitmap = uint(_committeeInfo.committeeBitmap);
+		uint i = 0;
+		MemberData memory md;
+		while (bitmap != 0) {
+			if (bitmap & 1 == 1) {
+				addrs[i] = participants[i];
```

Comment:
BUG - participants[i] should use a different index that advances with each loop iteration